contracts.go

```diff
@@ -224,3 +232,55 @@ func compileContract(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func reviseContract(cmd *cobra.Command, args []string) error {
+
+	// read the params file
+	params_b, err := ioutil.ReadFile("params.toml")
+	if err != nil {
+		return err
+	}
+
+	// read the contract template
+	template_b, err := ioutil.ReadFile("template.md")
+	if err != nil {
+		return err
+	}
+
+	// read history file
+	history_b, err := ioutil.ReadFile("history.toml")
+	if err != nil {
+		return err
+	}
+
+	// combine params, history, and template data
+	all_b := [3][]byte{params_b, template_b, history_b}
+	byte_array := make([]byte, 3, 3)
+
+	for _, element := range all_b {
+		byte_array = append(byte_array, element...)
+	}
```

Comment:
doesn't work because each element in all_b is itself a byte array, and append requires individual bytes. with your formula it would have to be something akin to `byte_array = append(byte_array, all_b...(...))` (where the first elipses is all elements in all_b, and the second elipses is all elements in each element of all_b), but it wouldn't make much sense for such thing to be possible, so we need that for loop. lemme know if i'm missing anything, but i tried to make the change and it was a wrong type error.